flowzone.yml

```diff
@@ -2980,6 +3045,161 @@ jobs:
           secrets: ${{ toJSON(secrets) }}
           variables: ${{ toJSON(vars) }}
 
+  ###################################################
+  # AWS/CloudFormation
+  ###################################################
+
+  cloudformation_test:
+    name: Test CloudFormation
+    runs-on: ${{ fromJSON(inputs.runs_on) }}
+    strategy:
+      fail-fast: true
+      matrix:
+        stack: ${{ fromJSON(needs.is_cloudformation.outputs.stacks) }}
+    timeout-minutes: ${{ fromJSON(inputs.jobs_timeout_minutes) }}
+    needs:
+      - is_pr_open
+      - is_cloudformation
+    if: |
+      !failure() && !cancelled() &&
+      needs.is_pr_open.result == 'success' &&
+      needs.is_cloudformation.outputs.cloudformation == 'true' &&
+      github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
If we are limiting CF deploys to internal PRs only, this logic can be moved to `is_cloudformation`